actuator/bindings/src/lib.rs

```diff
@@ -210,6 +222,37 @@ impl PyRobstrideActuator {
         })
     }
 
+    fn run_main_loop(&self, interval_ms: u64) -> PyResult<bool> {
+        self.rt.block_on(async {
+            let mut interval = time::interval(Duration::from_millis(interval_ms));
+            let supervisor_clone = self.supervisor.clone();
+            self.rt.spawn(async move {
+                loop {
+                    interval.tick().await;
+                    let mut clone: tokio::sync::MutexGuard<'_, Supervisor> =
+                        supervisor_clone.lock().await;
+                    if let Err(e) = clone.run_update_and_control().await {
+                        tracing::error!("Error in run_update_and_control: {:?}", e);
+                    }
+                    drop(clone);
+                }
+            });
+            return Ok(true);
+        })
+    }
```

Comment:
nit: add a newline between these
```suggestion
    }

```